namer/dnssrv/src/integration/scala/io/buoyant/namer/dnssrv/DnsSrvNamerIntegrationTest.scala

```diff
@@ -0,0 +1,21 @@
+package io.buoyant.namer.dnssrv
+
+import com.twitter.finagle.stats.NullStatsReceiver
+import com.twitter.finagle.{Addr, Name, NameTree, Path}
+import com.twitter.util.{Activity, Duration, NullTimer}
+import io.buoyant.test.{Awaits, FunSuite}
+import org.scalatest.Matchers
+import org.xbill.DNS
+
+class DnsSrvNamerIntegrationTest extends FunSuite with Awaits with Matchers {
+  test("can resolve some public SRV revord") {
+    val namer = new DnsSrvNamer(Path.empty, new DNS.ExtendedResolver, new NullTimer, Duration.Zero, new NullStatsReceiver)
+    Activity.sample(namer.lookup(Path.read("/_http._tcp.mxtoolbox.com."))) match {
```

Comment:
as mentioned in a previous comment, I'd appreciate a +1 that this shouldn't and doesn't just race on the `Activity` being in `Pending` state.